zproject/computed_settings.py

```diff
@@ -1130,3 +1132,5 @@ def zulip_path(path: str) -> str:
 THUMBOR_KEY = get_secret('thumbor_key')
 
 TWO_FACTOR_PATCH_ADMIN = False
+
+setup_sentry(SENTRY_DSN)
```

Comment:
What's the thinking about putting this here, in `computer_settings.py`?  I imagine it needs to run in some initialization code path, but I'm not sure this is the right place.

One thing to check is whether this has a startup performance impact, maybe testing via `time scripts/get-django-setting KEY`

Reply:
Sentry suggests putting it in `settings.py`, presumably so tit's loaded early enough: https://docs.sentry.io/platforms/python/django/

But excellent point, this does significantly affect the load timing, just because of the imports:
```
hyperfine --prepare 'echo' --prepare 'git checkout HEAD zproject/computed_settings.py' 'scripts/get-django-setting PRODUCTION  # Sentry' 'scripts/get-django-setting PRODUCTION  # No Sentry'
Benchmark #1: scripts/get-django-setting PRODUCTION  # Sentry
  Time (mean ± σ):     330.7 ms ±  11.0 ms    [User: 166.5 ms, System: 40.7 ms]
  Range (min … max):   321.8 ms … 355.2 ms    10 runs

Benchmark #2: scripts/get-django-setting PRODUCTION  # No Sentry
  Time (mean ± σ):     544.4 ms ±  10.5 ms    [User: 348.0 ms, System: 51.6 ms]
  Range (min … max):   525.1 ms … 561.2 ms    10 runs

Summary
  'scripts/get-django-setting PRODUCTION  # Sentry' ran
    1.65 ± 0.06 times faster than 'scripts/get-django-setting PRODUCTION  # No Sentry'
```

Reply:
By putting the `from .sentry import setup_sentry` behind a check of the value, we can get to:
```
Benchmark #1: scripts/get-django-setting PRODUCTION  # Sentry, disabled
  Time (mean ± σ):     266.8 ms ±  16.4 ms    [User: 231.6 ms, System: 32.3 ms]
  Range (min … max):   240.6 ms … 322.0 ms    25 runs

Benchmark #2: SENTRY_DSN=https://redacted@o48127.ingest.sentry.io/redacted scripts/get-django-setting PRODUCTION  # Sentry, enabled
  Time (mean ± σ):     629.1 ms ±  47.6 ms    [User: 568.5 ms, System: 53.6 ms]
  Range (min … max):   571.2 ms … 786.3 ms    25 runs

Benchmark #3: scripts/get-django-setting PRODUCTION  # No Sentry
  Time (mean ± σ):     268.5 ms ±  23.5 ms    [User: 231.7 ms, System: 33.9 ms]
  Range (min … max):   240.9 ms … 331.5 ms    25 runs

Summary
  'scripts/get-django-setting PRODUCTION  # Sentry, disabled' ran
    1.01 ± 0.11 times faster than 'scripts/get-django-setting PRODUCTION  # No Sentry'
    2.36 ± 0.23 times faster than 'SENTRY_DSN=https://redacted@o48127.ingest.sentry.io/redacted scripts/get-django-setting PRODUCTION  # Sentry, enabled'
```

Reply:
OK.  I imagine this doesn't affect our overall import time much, since it's just the `sentry-sdk` package we're adding to requirements (so all the other stuff we were likely importing ourselves later on).